log4j-perf-test/pom.xml

```diff
@@ -144,6 +142,28 @@
   <build>
     <plugins>
 
+      <!--
+        ~ Unban Logback.
```

Comment:
Unban or ban? The id says ban?

Reply:
Unban, the `<bannedDependencies>` rule is counter-intuitive.
The banned deps are listed as `excluded`, the deps that are allowed as `included`.

Reply:
The `id` of the rule must be the same as in `log4j-parent/pom.xml`.